### PR TITLE
improving language and clarity (#3274 )

### PR DESCRIPTION
**Pull request by hack3r-0m**
fixing [#3274](https://github.com/openebs/openebs/issues/3274)

### CODE REVIEW

File: docs/uninstall.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The recommended steps to uninstall the OpenEBS cluster gracefully is as follows.
   kubectl get sc
   ```
 
-- Delete the OpenEBS namespace either via `helm delete <chart name> --purge` if helm version is v2 or `helm uninstall <chart name> -n <namespace>` if helm version is v3 or `kubectl delete ns openebs` or you can delete the corresponding `openebs-operator` YAML using `kubectl delete -f <openebs-operator.yaml>`. You can check the status of OpenEBS namespace using the following command.
+- Delete the OpenEBS namespace using helm either via `helm delete <chart name> --purge` (if helm version is v2) or `helm uninstall <chart name> -n <namespace>` (if helm version is v3). It can aslo be deleted using kubectl as  `kubectl delete ns openebs` or you can delete the corresponding `openebs-operator` YAML using `kubectl delete -f <openebs-operator.yaml>`. You can check the status of OpenEBS namespace using the following command.
```
Comment (ranjithwingrider) [inline docs/uninstall.md:57]: Change aslo  -> also

**Review by ranjithwingrider — CHANGES_REQUESTED** [review]

File: docs/uninstall.md (hunk 1 above)
Comment (hack3r-0m) [reply docs/uninstall.md]: thanks for pointing out @ranjithwingrider :). Done

**Review by ranjithwingrider — APPROVED** [review]
Thank you @hack3r-0m for the changes...

**Comment by ranjithwingrider** [comment]
Looks like DCO is missing. Can you resolve that one? @hack3r-0m

**Comment by hack3r-0m** [comment]
I have signed DCO

**Pull request merged**